### DISCUSSION TITLE
Serverless / Edge Functions

### DISCUSSION THREAD

**Discussion by aabdulahad** [vercel/community#1667] [Help] [answered]
### Summary

[Vercel Limitations](https://vercel.com/docs/concepts/limits/overview#serverless-functions-created-per-deployment)

Using the above link - it states a hobby user may create 12 serverless functions per deployment for a Hobby account, made unlimited by switching to a Pro account. 
Is there any such limitation for edge functions created with NextJS 13 API routes?
Thanks in advance.

**Comment by leerob** [comment, marked as answer]
There is no limit on the number of Edge Functions 👍 

See other limits here: https://vercel.com/docs/concepts/functions/edge-functions/limitations

**Reply by aabdulahad** [reply]
Thank you. 👍
